forms-flow-web/src/components/Form/Item/Edit.js

```diff
@@ -223,6 +223,12 @@ const Edit = React.memo((props) => {
     
   };
 
+// setting the main option details to the formdata
+  const handleChange = (path, event) => {
+    const {target} = event;
+    const value = target.type === 'checkbox' ? target.checked : target.value;
+    dispatchFormAction({type: path, value});
+  
   // setting the main option details to the formdata
```

Comment:
Seems like `handleChange` is duplicated.

Reply:
good catch, change is added to my current commit for fixing build issues